src/plugin/helpers/element.ts

```diff
@@ -115,13 +134,28 @@ export const createGetElementDataFromNode = <
       return;
     }
 
-    values[fieldName] = value;
+    if (fieldDescription.type === "repeater") {
+      values[fieldName] = values[fieldName]
+        ? (values[fieldName] as unknown[]).concat(value)
+        : [value];
+
+      console.log({ value, values: values[fieldName] });
```

Comment:
Do we still want this `console.log` here?

Reply:
Ta for spotting, removed.